src/index.js

```diff
@@ -165,6 +165,8 @@ module.exports = function (babel) {
                 ])
               );
             } else {
+              path.scope.crawl();
```

Comment:
I worry that doing this for **every** import declaration replacement is going to be quite costly. Now that you fully understand what is wrong, I wonder if we can avoid the re-crawl by detecting the specific failure scenario and handling it. 🤔 

Thoughts @mdeanjones?

Reply:
Makes sense. I opted for a "_try, and try again_" approach. It gets the job done, but does feel a bit... hacky.